.travis.yml

```diff
@@ -28,9 +28,10 @@ install:
 
 # command to run tests
 script:
-  - flake8 . --exclude=.venv,.build,foo --ignore=E501,F403,E402
-  - nosetests --with-doctest --ignore-files=plotCoverage deeptools
-  - nosetests deeptools/test/test_plotCoverage.py
+  - flake8 . --exclude=.venv,.build,foo,build,deeptoolsintervals/__init__.py --ignore=E501,F403,E402
```

Comment:
`build/` is created with `python ./setup.py install`. `foo/` is a venv for the planemo tests.